jsonrpsee/Cargo.toml

```diff
@@ -20,6 +20,6 @@ types = { path = "../types", version = "0.2.0", package = "jsonrpsee-types", opt
 
 [features]
 client = ["http-client", "ws-client"]
-server = ["http-server", "ws-server", "utils"]
+server = ["http-server", "ws-server", "utils", "macros"]
```

Comment:
I think types should be sufficient here, no?

Reply:
"types" is sufficient, but I'm a bit concerned that [this](https://github.com/paritytech/jsonrpsee/blob/dd4a17ba5e92091458805ad8017c2926cd28d1fa/jsonrpsee/src/lib.rs#L28-L35) will become messy. I'd like to be able to look at `lib.rs` and see at a glance where everything is coming from.